src/gatsby-plugin-chakra-ui/theme.js

```diff
@@ -20,6 +20,9 @@ const customTheme = {
     '2xl': '2rem',
     '3xl': '2.5rem',
   },
+  // Mobile views were not covered under default 'sm' breakpoint
+  // Going to just write in mock sizes, but open to changing them
+  breakpoints: ['16em', '48em', '64em', '80em'],
```

Comment:
This might be a good one to ask about in design channels if you haven't already

Reply:
@mbifulco I think we're cool to just use the ol' create a breakpoint once things start looking broken, methodology for now! Just to keep things rolling. We can refactor to standardize breakpoints if we need.